src/ssl.c

```diff
@@ -41022,6 +41022,16 @@ int  wolfSSL_get_chain_cert_pem(WOLFSSL_X509_CHAIN* chain, int idx,
 #endif /* WOLFSSL_PEM_TO_DER || WOLFSSL_DER_TO_PEM */
 }
 
+/* get session IDSz */
+int wolfSSL_get_sessionIDSz(const WOLFSSL_SESSION* session)
```

Comment:
Tough having a new API just for size. I checked and `wolfSSL_get_sessionID` is a non-standard API (not in openssl). 

Perhaps it would make sense to add an _ex style API for getting session id? Something like this:

```
int wolfSSL_get_sessionID_ex(const WOLFSSL_SESSION* session, const byte** id, int* idSz)
{
    int ret = BAD_FUNC_ARG, sz = 0;
    if (session != NULL) {
        sz = session->sessionIDSz;
        if (id)
            *id = session->sessionID;
        ret = WOLFSSL_SUCCESS;
    }
    if (idSz)
        *idSz = sz;
    return ret;
}
```